meson.build

```diff
@@ -208,7 +208,7 @@ endif
 libjcat = dependency('jcat', version : '>= 0.1.0', fallback : ['libjcat', 'libjcat_dep'])
 libjsonglib = dependency('json-glib-1.0', version : '>= 1.1.1')
 valgrind = dependency('valgrind', required: false)
-soup = dependency('libsoup-2.4', version : '>= 2.51.92')
+libcurl = dependency('libcurl', version : '>= 7.62.0')
```

Comment:
Can it be older or is this just the first version you tried?

Reply:
It needs to be that for the CURLU support.